scripts/promote-stable-rc

```diff
@@ -3,7 +3,7 @@
 set -ex
 
 channel="stable-rc"
-read version <<< $(git describe --tags | tr -d "v")
+version=$(node -p -e "require('./package.json').version")
```

Comment:
@RasPhilCo I realize this is kinda late now but I was looking at the node docs and it seems like `-e` is redundant here.
https://nodejs.org/api/cli.html#cli_p_print_script